integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDynamicUpdate.java

```diff
@@ -443,7 +452,61 @@ public void testMiiRemoveTarget() {
 
   }
 
+  /**
+   * Negative test: Changing the domain name using mii dynamic update.
+   */
+  @Test
+  @Order(5)
+  @DisplayName("Negative test changing domain name using mii dynamic update")
+  public void testMiiChangeDomainName() {
+
+    // This test uses the WebLogic domain created in BeforeAll method
+    // BeforeEach method ensures that the server pods are running
+
+    // write sparse yaml to file
+    Path pathToChangeDomainNameYaml = Paths.get(WORK_DIR + "/changedomainname.yaml");
+    String yamlToChangeDomainName = "topology:\n"
+        + "  name: newdomainname\n"
+        + "  AdminServerName: 'admin-server'";
 
+    assertDoesNotThrow(() -> Files.write(pathToChangeDomainNameYaml, yamlToChangeDomainName.getBytes()));
+
+    // Replace contents of an existing configMap
+    replaceConfigMapWithModelFiles(configMapName, domainUid, domainNamespace,
+        Arrays.asList(pathToChangeDomainNameYaml.toString()), withStandardRetryPolicy);
+
+    // Patch a running domain with introspectVersion.
+    patchDomainResourceWithNewIntrospectVersion(domainUid, domainNamespace);
+
+    // Verifying introspector pod is created and failed
+    logger.info("verifying the introspector failed and the pod log contains the expected error msg");
+    String expectedErrorMsg = "name is not one of the attribute names allowed in model location topology";
```

Comment:
I don't think this is the correct message.  Are you wait for the latest WDT release 1.9.8 (not yet released but should be today).

Reply:
I tried with latest WDT from https://objectstorage.us-phoenix-1.oraclecloud.com/n/weblogick8s/b/wko-system-test-files/o/weblogic-deploy-master.zip. I got the same error msg. What's the new error msg I should get?